src/RecoveryServices/RecoveryServices.SiteRecovery/Fabrics/StartAzureRmRecoveryServicesAsrSwitchProcessServerJob.cs

```diff
@@ -24,6 +25,7 @@ namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery
     [Cmdlet("Start", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "RecoveryServicesAsrSwitchProcessServerJob",DefaultParameterSetName = ASRParameterSets.Default,SupportsShouldProcess = true)]
     [Alias("Start-ASRSwitchProcessServerJob")]
     [OutputType(typeof(ASRJob))]
+    [GenericBreakingChange("Property type ASRHealthError_2016_08_10 changed to ASRHealthError")]
```

Comment:
Why is this necessary?  If this is just a change in namespace, then there is no need for a breaking change entry.  If there are changed properties, you need to describe them more thoroughly.  This applies to all similar changes below.